roles/uperf/templates/server.yml.j2

```diff
@@ -57,6 +57,18 @@ items:
             - name: net.ipv4.ip_local_port_range
               value: 20000 20011
 {% endif %}
+{% if workload_args.tuned is sameas true %}
+          serviceAccountName: benchmark-operator
+          securityContext:
+            privileged: true
+            sysctls:
+            - name: net.ipv4.ip_local_port_range
+              value: 1024 65535
+            - name: net.ipv4.tcp_tw_reuse
+              value: "1"
+            - name: net.netfilter.nf_conntrack_tcp_timeout_time_wait
+              value: "1"
```

Comment:
I think this sysctl parameters won't work on this context as they are not namespaced in the Linux Kernel and can't be set at container level.
You should set them using node tuning operator with something similar to:

```yaml
apiVersion: tuned.openshift.io/v1
kind: Tuned
metadata:
  name: openshift-ingress-performance
  namespace: openshift-cluster-node-tuning-operator
spec:
  profile:
  - data: |
      [main]
      summary=Network performance profile
      include=openshift
      [sysctl]
      net.ipv4.ip_local_port_range="1024 65535"
      net.ipv4.tcp_tw_reuse=1
      net.netfilter.nf_conntrack_tcp_timeout_time_wait=1
    name: openshift-ingress-performance
  recommend:
  - match:
    - label: {node-role.kubernetes.io/workload: node-role.kubernetes.io/worker
    priority: 10
    profile: openshift-network-performance
```

Reply:
Hi @rsevilla87 !
Yes, those node tuning operator templates are certainly needed, but unfortunately they are not enough for this scenario, you need both (the tuning operator templates and the pod level sysctl adjustments). Otherwise, you will get the sysctl tuning applied to the workers and not into the pods (which we need for this specific testing).
I agree with you on that creating the tuned profile (and the corresponding kubelet configuration for this scenario) should be done outside benchmark-operator, and that is why we created the `tuned` parameter, that should only be enabled once proper configurations outside the scope of the operator are in place.